TOPSPIN/Phase_Correction/4D.md

```diff
@@ -1,36 +1,286 @@
-# Phase Correction in 4D Spectra
+# Phase Correction in 4D Protein Spectra
 
-Phase correction in 4D spectra follows a similar workflow as in 2D spectra. The process involves performing NUS reconstruction, extracting projections, and then phase-correcting them.
+The following tutorial refers to the **4D HCNH NOESY** spectrum type but the same procedure can be applied for other 4D spectra.
+Specifically for **4D HCNH NOESY**, we need to phase only the **F4(PCH0)** value. There are several ways to do that.
 
-## Steps for Phase Correction
+----------------------------------------------------------------------
+## Solution 1: Phase-Correct the 1st FID
+
+**Example Data:** Raw_Spectra/Carbonic_Anhydrase_2/hCA2-wt_850/9
+
+1. **Reset all F4 phase values to `0`:**
+   > **Important:** Look at the bottom of the pulse program to find the correct PHC0 and PHC1 values of F1-F3 - if 
+there are any. Read the [NUS-reconstruction tutorials](../../Non_Uniform_Sampling/Bruker_NUS/4D.md) for details. Briefly the pulse programs 
+`hsqcnoesyhsqccngp4d` and `noehcnhwg4d_nove` require zero values, while `sfhmqc_noe_sfhmqc_4Dhcnh.fl` the following:
+ 
+   |   |F4 |F3 |F2 |F1 |
+   |---|---|---|---|---|
+   |PHC0|<P0>|90|90|-45|
+   |PHC1|0  |-180|-180|0  |
+
+Where `P0` is the value you will find after phase correction.
+
+   Unless you use `sfhmqc_noe_sfhmqc_4Dhcnh.fl`, set all phase values to zero with the following command: 
+```
+4 PHC0 0
+4 PHC1 0
+3 PHC0 0
+3 PHC1 0
+2 PHC0 0
+2 PHC1 0
+1 PHC0 0
+1 PHC1 0
+```
+
+2. **Initiate NUS-reconstruction with zero phase values:**
+   
+   Follow the instructions at [NUS-reconstruction tutorials](../../Non_Uniform_Sampling/Bruker_NUS/4D.md). Once all 
+parameters are set correctly nad the phase angles are zero as explained above, issue `ftnd 0`. 
+
+3. **Extract the 1st FID:**  
+   Open the processed 4D spectrum and run `rser eao 1` to extract the 1st FID.
+
+4. **Switch to the Frequency Domain:**  
+   Use `ft` to switch to the frequency domain.
+
+5. **Manual Phase Correction:**  
+   Enter `.ph` for manual phase correction. Set the pivot line (right mouse click) near the left limit end to modify only `PHC0`, not `PHC1`.  
+   > **Note:** Adjusting `PHC1` (1st order correction) has minimal effect on frequencies near the pivot and no effect on `PHC0`.  
+   ![before phasing](images/CA2_850_1.fid_before_phasing.png)
+
+6. **Adjust PHC0:**  
+   Adjust only `PHC0` (0th order correction) so the FID line extends straight and the highest peaks point upwards.  
+   > **Note:** Avoid using automatic phasing (`apk` and its variants) as it generally doesn't work well.  
+   ![after phasing](images/CA2_850_1.fid_after_phasing.png)
+
+7. **Save and Apply PHC0:**  
+   Note down the optimal `PHC0` value (e.g., `-115.0` degrees). Click the **Save-disk nD** icon to copy the values to 
```

Comment:
*Optional correction*: I would do it with saving the phased 1D:

"Click the **Save-disk nD** icon to copy the values to the 4D spectrum, then click the **Save and Return icon**. Issue `PHC0` and note down the value (here, -115.0 degrees). <...>"